ramls/eholdings.raml

```diff
@@ -7,6 +7,109 @@ documentation:
   - title: mod-kb-ebsco (category)
     content: Implements the eholdings interface using the EBSCO kb as a backend.
 
+traits:
+  indexable:
+    queryParameters: 
+      q:
+        displayName: Query
+        type: string
+        description: Text entered in search field
+        example: Basket Weaving
+        required: true
+      page:
+        displayName: Page
+        type: number
+        description: Page offset from which results are retrieved
+        example: 1
+        required: false
+      count:
+        displayName: Count
+        enum: [10, 20, 30]
+        description: The maximum number of results to return in the response.
+        example: 10
+        required: false
+      sort:
+        displayName: Sort options
+        enum: [name, relevance]
+        description: Option by which results are sorted
+        default: relevance
+        required: false
+  filterable:
+    queryParameters:
+      filter[selected]:
+        displayName: Selection status
+        enum: [true, false, ebsco]
```

Comment:
at least in packages, we support [true, false, ebsco, all]; is that different for providers?

Reply:
The way I understood this one when I was playing around with it, selecting `all` just causes this query parameter to be omitted from the request.